biodatasets/n2c2_2018/n2c2_2018_track2.py

```diff
@@ -0,0 +1,427 @@
+# coding=utf-8
```

Comment:
can we make the directory name match the dataset name to be consistent with the other dataset loaders? 
`biodatasets/n2c2_2018_track2/n2c2_2018_track2.py`

Reply:
I intentionally did not do that to accomodate for #490 so both tracks come under single directory. It works well with tests as well.

Reply:
yes, I understand. I think it's better to follow the pattern that already exists in the repository though. for example in biodatasets, 
```
├── tmvar_v1
│   └── tmvar_v1.py
├── tmvar_v2
│   └── tmvar_v2.py
```

```
├── n2c2_2006_deid
│   └── n2c2_2006_deid.py
├── n2c2_2006_smokers
│   └── n2c2_2006_smokers.py
```

```
├── bioasq_task_b
│   └── bioasq_task_b.py
├── bioasq_task_c_2017
│   └── bioasq_task_c_2017.py
```

Reply:
Thanks, I've updated as suggested.